packages/storybook/src/theme.ts

```diff
@@ -227,6 +229,12 @@ const buttons = {
   },
 }
 
+const selects = {
+  selectMenu: {
+    top: `calc(100% + ${space[2]}px)`,
```

Comment:
What do you think about splitting these two out?
```suggestion
    marginTop: 2,
    top: `100%`,
```

Reply:
This is totally up to the theme so it can be either or.